libraries/chain/include/muse/chain/config.hpp

```diff
@@ -92,7 +92,7 @@
 #define MUSE_MAX_TIME_UNTIL_EXPIRATION       (60*60) // seconds,  aka: 1 hour
 #define MUSE_MAX_MEMO_SIZE                   2048
 #define MUSE_MAX_PROXY_RECURSION_DEPTH       4
-#define MUSE_VESTING_WITHDRAW_INTERVALS      13
+#define MUSE_VESTING_WITHDRAW_INTERVALS      4 //changing vesting period from 13 weeks to 4 weeks 
```

Comment:
See above: you cannot simply change this because the change would happen retroactively. You can define a new interval, and use one or the other depending on whether it happens before or after the upgrade.